File: Code/Websites/DanpheEMR/Services/Emergency/DTOs/ERGetConsentFormDTO.cs
using System;

namespace DanpheEMR.Services.Emergency.DTOs
{
    public class ERGetConsentFormDTO
    {
        /// <summary>
        /// ERPatientId refers to the Emergency Patient for which transaction is being done
        /// </summary>
        public int ERPatientId { get; set; }
        /// <summary>
        /// FileId refers to the Consent File
        /// </summary>
        public int FileId { get; set; }
        /// <summary>
        /// PatientId refers to the Patient for which transaction is being done
        /// </summary>
        public int PatientId { get; set; }
        /// <summary>
        /// ShortName refers to Patient's ShortName
        /// </summary>
        public string ShortName { get; set; }
        /// <summary>
        /// FileType refers to the type of Consent File
        /// </summary>
        public string FileType { get; set; }
        /// <summary>
        /// FileName refers to the Consent File Name
        /// </summary>
        public string FileName { get; set; }
        /// <summary>
        /// DisplayName refers to the Consent File DisplayName
        /// </summary>
        public string DisplayName { get; set; }
        /// <summary>
        /// BinaryData refers to the Binary Data of the Consent File
        /// </summary>
        public string BinaryData { get; set; }
    }
}

File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/emergency/shared/dto/er-get-consent-form.dto.ts
/**
 * @summary This dto is to represent the response received from the server while fetching Consent Form information
 */
export class ERGetConsentForm_DTO {
    /**
     * @summary ERPatientId refers to the Emergency Patient for which transaction is being done
     */
    ERPatientId: number = 0;
    /**
     * @summary FileId refers to the Consent File
     */
    FileId: number = 0;
    /**
     * @summary PatientId refers to the Patient for which transaction is being done
     */
    PatientId: number = 0;
    /**
     * @summary ShortName refers to Patient's ShortName
     */
    ShortName: string = "";
    /**
     * @summary FileType refers to the type of Consent File
     */
    FileType: string = "";
    /**
     * @summary FileName refers to the Consent File Name
     */
    FileName: string = "";
    /**
     * @summary DisplayName refers to the Consent File DisplayName
     */
    DisplayName: string = "";
    /**
     * @summary BinaryData refers to the Binary Data of the Consent File
     */
    BinaryData: string = "";
}